[2022-10-06 05:02:43,611] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2022-10-03T00:00:00+00:00 [queued]>
[2022-10-06 05:02:43,640] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2022-10-03T00:00:00+00:00 [queued]>
[2022-10-06 05:02:43,646] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-10-06 05:02:43,648] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-10-06 05:02:43,648] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-10-06 05:02:43,695] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-10-03 00:00:00+00:00
[2022-10-06 05:02:43,712] {standard_task_runner.py:52} INFO - Started process 119 to run task
[2022-10-06 05:02:43,730] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2022-10-03T00:00:00+00:00', '--job-id', '324', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp955fst7_', '--error-file', '/tmp/tmp5fe_6275']
[2022-10-06 05:02:43,732] {standard_task_runner.py:77} INFO - Job 324: Subtask local_to_gcs_task
[2022-10-06 05:02:43,889] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.local_to_gcs_task scheduled__2022-10-03T00:00:00+00:00 [running]> on host d1144214b887
[2022-10-06 05:02:43,988] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-10-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-03T00:00:00+00:00
[2022-10-06 05:02:47,379] {python.py:175} INFO - Done. Returned value was: None
[2022-10-06 05:02:47,405] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20221003T000000, start_date=20221006T050243, end_date=20221006T050247
[2022-10-06 05:02:47,516] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-10-06 05:02:47,608] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
